app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<h2>Amy's Task list</h2>
+
+<h3><%= link_to "Add Task", new_task_path %></h3>
+
+<ul>
+<% @tasks.each do |task| %>
+  <li>
+    NAME: <%= task.name %>, 
+    DESCRIPTION: <%= task.description %>,
+    COMPLETION DATE: <%= task.complete_date %>
+    <br>
+    <%= link_to "View Task", task_path(task.id) %>
+    <% if task.complete == false %>
```

Comment:
Here, you are checking specifically if `task.complete` is equal to `false`. However, the default value for `task.complete` for a new task ends up being `nil`. Therefore, this conditional ends up incorrectly displaying